vaadinfrontend/frontend/themes/datamanager/components/card.css

```diff
@@ -146,3 +146,63 @@
 .experiment-info .title {
   font-weight: bold;
 }
+
+.card.experimental-variables {
+  flex: auto;
+  font-size: var(--lumo-font-size-s);
+  padding: var(--lumo-space-l);
+}
+
```

Comment:
```suggestion
.card.experimental-variables {
  flex: auto;
  font-size: var(--lumo-font-size-s);
  padding: var(--lumo-space-l);
  margin: var(--lumo-space-s);
}
```
I suggest adding a margin so the cards are not stuck together. I like how the small margin looks, but previously it was larger (see testing instance).

Reply:
You would need to add the same to
`.card.experiment-info`
further up, so they are aligned.

Reply:
That's the margin between variables: 
<img width="839" alt="image" src="https://github.com/qbicsoftware/data-manager-app/assets/10232219/aaa1c513-f4e8-46a7-8707-7f260b207773">
The margin between both cards is missing.

Reply:
The margin between components will be addressed in an own PR, since we need refactor the complete Experimental Page anyway. I work bottom up: first the individual pieces and orchestrate everything at the end :)